doc/repository_format/introduction.rst

```diff
@@ -0,0 +1,37 @@
+.. _specs_misc_ref-label:
+
+#########################
+yum/dnf Repository Format
+#########################
+
+There are now a good handful of package managers that have used and evolved
+a repository format that is generally referred to as a `yum` repository. The
+original release of `yum`, the "Yellowdog Updater Modified", was in 2002,
+and was present in Fedora Core 1, released in 2003.
+
+This documentation has the goal of documenting what modern tooling should
+generate for `dnf5`, but also cover the quirks of in-the-wild repositories.
+
+Since the repository format is primarily XML based, XML Schema Definition (XSD)
+files will be used so that validation can be performed.
+
+Tools parsing the format should accept the permissive variants of the schema
+definitions, while tools creating the format should produce the strict variants.
+
+History
+=======
+
+There have been previous attempts to document the repository format, most
+notably in openSUSE.
+https://en.opensuse.org/openSUSE:Standards_Rpm_Metadata
+
+There exist repositories using a SQLLite variant of the repository format,
+with some repositories *exclusively* using it, having only the `repomd.xml`
+top level file in XML. The SQLLite variant is understood by `yum`, but did
+not make the leap to `dnf` at all.
```

Comment:
The SQLite variant is not understood by libsolv, and that's why most current-generation RPM package managers don't support it.

See: https://github.com/openSUSE/libsolv/issues/92

Reply:
Added a snipped on this, but didn't add a link to the issue.. but could do so.